eventbrite.py

```diff
@@ -0,0 +1,380 @@
+      "auto_hide_after": "",
+      "sales_channels": ["online","atd"],
+      "hide_sale_dates": false,
+      "delivery_methods": ["electronic"],
+      "include_fee": false,
+      "sales_start": "2023-11-09T08:00:00Z",
+      "sales_end": "2023-11-11T03:00:00Z"
+    }
+}
+"""
+
+def getOrganization(http):
+    url = f"{EVENTBRITE_API_BASE_URL}users/me/organizations/"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status == 200:
+        data = resp.data.decode('utf-8')
+        print(data)
+    else:
+        print(f"HTTP Error: {resp.status}")
+
+def getEventById(http, event_id):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status == 200:
+        data = resp.data.decode('utf-8')
+        print(data)
+    else:
+        print(f"HTTP Error: {resp.status}")
+
+def getEventsByOrganization(http, organization_id):
+    url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status == 200:
+        data = resp.data.decode('utf-8')
+        print(data)
+    else:
+        print(f"HTTP Error: {resp.status}")
+
+def createEvent(http, organization_id ,event_details):
+    url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events/"
+    headers = {
+    'Authorization': eventbrite_api_token,
+    'Content-Type': 'application/json'
+    }
+
+    try:
+        # Make the POST request
+        resp = http.request('POST', url, headers=headers, body=event_details)
+        # Check the status code to ensure a successful request
+        if resp.status == 200:
+            data = resp.data.decode('utf-8')
+            print(data)
+            return json.loads(data)
+        elif resp.status == 400:
+            error_data = json.loads(resp.data.decode('utf-8'))
+            error_detail = error_data.get('error_detail', {})
+            return {
+                "error": error_data.get('error', 'Unknown Error'),
+                "error_description": error_data.get('error_description', 'An error occurred'),
+                "error_detail": error_detail,
+                "status_code": resp.status
+            }
+        else:
+            print(f"HTTP Error: {resp.status}")
+            return None
+    except urllib3.exceptions.HTTPError as e:
+        return {
+            "error": "HTTP Error",
+            "error_description": str(e),
+            "status_code": e.code
+        }
```

Comment:
Create a `quick_create_event` function that calls `createEvent`, `createTicketClass` and `publishEvent`:
```py
def quick_create_event(http, organisation_id, title, description, cost_cents=None, publish=True):
  ticket_detail = {
    "ticket_class": {
      "name": "API Test0",
      "free": true,
      "donation": false,
      "quantity_total": "100",
      "minimum_quantity": "1",
      "maximum_quantity": "10",
      "hidden": false,
      "auto_hide": false,
      "auto_hide_before": "",
      "auto_hide_after": "",
      "sales_channels": ["online","atd"],
      "hide_sale_dates": false,
      "delivery_methods": ["electronic"],
      "include_fee": false,
      "sales_start": "2023-11-09T08:00:00Z",
      "sales_end": "2023-11-11T03:00:00Z",
      **({"cost": f"AUD,{cost_cents}"} if cost is not None else {}),
    }
}
```

Reply:
Also need a `date_start` argument. I want to hardcode the venue to be hillview intercultural center https://www.google.com/maps/place/Hillview+Intercultural+Community+Centre/@-32.0090192,115.9003645,17z/data=!3m1!4b1!4m6!3m5!1s0x2a32bdcd5da2c777:0x2fbd272e005ce3d5!8m2!3d-32.0090238!4d115.9029448!16s%2Fg%2F11pckj99n5?entry=ttu